src/plugins/intel_gna/src/transformations/insert_copy_layer.cpp

```diff
@@ -19,6 +19,7 @@ using namespace ov::intel_gna::ngraph_util;
 
 NGRAPH_RTTI_DEFINITION(InsertCopyBeforeAssignLayer, "InsertCopyBeforeAssignLayer", 0);
 NGRAPH_RTTI_DEFINITION(InsertCopyBeforeConcatLayer, "InsertCopyBeforeConcatLayer", 0);
+NGRAPH_RTTI_DEFINITION(InsertCopyBeforeBroadcastLayer, "InsertCopyBeforeBroadcastLayer", 0);
```

Comment:
Please added detailed description of the new transformation in the header (like it done for other transformations)